contracts/Coordinator.sol

```diff
@@ -28,7 +28,7 @@ contract Coordinator {
             address(dxMgnPool1.secondaryToken())
         );
         // Since both auctions start at the same time, it suffices to check one.
-        return auctionIndex > dxMgnPool1.lastParticipatedAuctionIndex();
+        return auctionIndex > dxMgnPool1.lastParticipatedAuctionIndex() && uint(dxMgnPool1.currentState()) == 0;
```

Comment:
Is it not possible to use the enum state here? If we define it as public somehow? It's error prone to use hard coded integers.

Could we alternatively expose this functionality as a public view method directly on the pool?

Reply:
I tried to find a solution. I tried to make it public, but I got a compiler error immediately.

But, yeah, exposing it via a view method is a good alternative. We should do it.

Reply:
Okay, it was really easy. I should have google more...